deepfence_server/handler/scan_reports.go

```diff
@@ -218,17 +219,30 @@ func (h *Handler) StartComplianceScanHandler(w http.ResponseWriter, r *http.Requ
 		return
 	}
 
-	var scanTrigger model.ComplianceScanTrigger
-	if len(reqs.ScanTriggers) > 0 {
-		scanTrigger = reqs.ScanTriggers[0]
+	ctx := r.Context()
+
+	var nodes []model.NodeIdentifier
+	if len(reqs.NodeIds) == 0 {
```

Comment:
This logic has changed entirely, please have a look [here](https://github.com/deepfence/ThreatMapper/commit/5d4994c6713cfd67d62a54735282c15dbd0d9e39#diff-0f794619c2000b03dc230ee35f9410c6b5aa7d271444f77d91c249a568022bbfR855)